AmplifyTools/amplify-tools.sh

```diff
@@ -29,66 +38,65 @@ AMP_CLI_VERSION_CURRENT=`npx -q amplify --version |tail -1`
 AMP_CLI_VERSION_MINIMUM="4.21.0"
 AMP_CLI_VERSION_INVALID=0
 
-
 STRIP_ESCAPE_RESULT=
 stripEscapeUtil() {
-    STRIP_ESCAPE_RESULT=
+  STRIP_ESCAPE_RESULT=
 
-    set +e
-    HAS_RUBY=0
-    which ruby > /dev/null 2>&1
-    if [[ $? -eq 0 ]]; then
-	HAS_RUBY=1
-    fi
-    set -e
+  set +e
+  HAS_RUBY=0
+  which ruby >/dev/null 2>&1
+  if [[ $? -eq 0 ]]; then
+    HAS_RUBY=1
+  fi
+  set -e
 
-    if [ ${HAS_RUBY} -eq 1 ]; then
-	STRIP_ESCAPE_RESULT=`echo "${1}" | ruby -pe 'gsub(/\e\[[0-9;]*m/s, "")'`
-    else
-	STRIP_ESCAPE_RESULT=`echo "${1}" | npx -q strip-ansi-cli`
-    fi
+  if [ ${HAS_RUBY} -eq 1 ]; then
+    STRIP_ESCAPE_RESULT=`echo "${1}" | ruby -pe 'gsub(/\e\[[0-9;]*m/s, "")'`
+  else
+	  STRIP_ESCAPE_RESULT=`echo "${1}" | npx -q strip-ansi-cli`
+  fi
 }
 
 VERSION_INVALID=
 checkMinVersionCompatibility() {
-    VERSION_INVALID=0
-
-    stripEscapeUtil "${1}"
-    CURR_VERSION_SAFE="${STRIP_ESCAPE_RESULT}"
-    CURR_VERSION_MAJOR=`echo "${CURR_VERSION_SAFE}" | tr  '.' ' ' | awk '{print $1}'`
-    CURR_VERSION_MINOR=`echo "${CURR_VERSION_SAFE}" | tr  '.' ' ' | awk '{print $2}'`
-    CURR_VERSION_PATCH=`echo "${CURR_VERSION_SAFE}" | tr  '.' ' ' | awk '{print $3}'`
-
-    stripEscapeUtil "${2}"
-    REQU_VERSION_SAFE="${STRIP_ESCAPE_RESULT}"
-    REQU_VERSION_MAJOR=`echo "${REQU_VERSION_SAFE}" | tr  '.' ' ' | awk '{print $1}'`
-    REQU_VERSION_MINOR=`echo "${REQU_VERSION_SAFE}" | tr  '.' ' ' | awk '{print $2}'`
-    REQU_VERSION_PATCH=`echo "${REQU_VERSION_SAFE}" | tr  '.' ' ' | awk '{print $3}'`
-
-    if [ -z "${CURR_VERSION_MAJOR}" ] ||
-	[ -z "${CURR_VERSION_MINOR}" ] ||
-	[ -z "${CURR_VERSION_PATCH}" ] ||
-	[ -z "${REQU_VERSION_MAJOR}" ] ||
-	[ -z "${REQU_VERSION_MINOR}" ] ||
-	[ -z "${REQU_VERSION_PATCH}" ]; then
-	VERSION_INVALID=1
-	return
-    fi
+  VERSION_INVALID=0
+
+  stripEscapeUtil "${1}"
```

Comment:
Is it possible to revert the space changes in this file?

Reply:
it is, let me do it. Sorry, only realized to late that I had "format on save" for `.sh` files

Reply:
done